src/include/add-on/add-on-workflow.rb

```diff
@@ -1390,11 +1390,13 @@ def AdjustRepositoryInfo(info)
         end
         source_data = Pkg.SourceGeneralData(one_repo)
         if source_data != nil && Builtins.haskey(source_data, "base_urls")
-          Ops.set(
-            info.value,
-            "URLs",
-            Ops.get_list(source_data, "base_urls", [])
-          )
+          urls = source_data["base_urls"]
```

Comment:
BTW for future it would be nice if we return object for that repository with methods like url_display.